limits.c

```diff
@@ -115,7 +115,7 @@ void limits_go_home(uint8_t cycle_mask)
   uint8_t approach = ~0;  //approach has all bits set (negative dir) or none (positive)
   float homing_rate;
   uint8_t idx;
-  uint8_t n_cycle = (2*N_HOMING_LOCATE_CYCLE+1);
+  uint8_t n_cycle = (2*N_HOMING_LOCATE_CYCLE);
```

Comment:
This is the 'use rising edge' change, refactored.  Instead of adding one and changing the loop condition, keep the loop condition and skip the add.